public/app/features/commandPalette/CommandPalette.tsx

```diff
@@ -21,23 +12,49 @@ import { EmptyState, Icon, LoadingBar, useStyles2 } from '@grafana/ui';
 import { t } from 'app/core/internationalization';
 
 import { KBarResults } from './KBarResults';
+import { KBarSearch } from './KBarSearch';
 import { ResultItem } from './ResultItem';
 import { useSearchResults } from './actions/dashboardActions';
-import useActions from './actions/useActions';
+import {
+  useRegisterRecentDashboardsActions,
+  useRegisterRecentScopesActions,
+  useRegisterScopesActions,
+  useRegisterStaticActions,
+} from './actions/useActions';
 import { CommandPaletteAction } from './types';
 import { useMatches } from './useMatches';
 
 export function CommandPalette() {
+  useRegisterStaticActions();
```

Comment:
These are now loaded before cmdK is opened but others which need network are loaded after it's open. It is a tradeoffs and we can move things around but as we add things it made more sense to me to have a way to do load both ways.